src/pages/InfoList/SortBox/SortBoxFilterForm.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import styled from 'styled-components';
+import DropdownButton from './DropdownButton';
+import SearchInput from './SearchInput';
+import SearchButton from './SearchButton';
+
+const SortBoxFilterForm = () => {
+  return (
+    <>
+      <FilterRow>
+        <>
+          <SortTitle>여행지/숙소</SortTitle>
+          <SearchInput />
+        </>
+        <CountryButtonContainer>
+          <DropdownButton className="dropdownButton">국내전체</DropdownButton>
+        </CountryButtonContainer>
+        <Checkinout>
+          <>
+            <SortTitle>체크인</SortTitle>
+            <DropdownButton>체크인</DropdownButton>
+          </>
+          <>
+            <SortTitle>체크아웃</SortTitle>
+            <DropdownButton className="checkout">체크아웃</DropdownButton>
+          </>
+        </Checkinout>
+      </FilterRow>
+      <FilterRow>
+        <DropdownButton>인원</DropdownButton>
+        <DropdownButton>가격 범위</DropdownButton>
+        <DropdownButton>스테이 유형</DropdownButton>
+      </FilterRow>
+      <SearchButtonContainer>
+        <SearchButton />
+      </SearchButtonContainer>
+    </>
+  );
+};
+
+const FilterRow = styled.div`
+  display: flex;
+  align-items: center;
+  border-bottom: 1px solid #e6e6e6;
+  height: 60px;
+  position: relative;
+`;
+
+const SortTitle = styled.span`
+  font-size: 17px;
+  color: ${props => props.theme.fontColor};
```

Comment:
```suggestion
  color: ${({ theme }) => theme.fontColor};
```
이 자리에 들어가는게 결국 함수이기 때문에 이 부분도 구조분해 할당 가능합니다

Reply:
넵 해당 부분 구조분해 할당하도록 하겠습니다!